src/main/kotlin/no/nav/medlemskap/domene/personhistorikk/Statsborgerskap.kt

```diff
@@ -21,6 +21,15 @@ data class Statsborgerskap(
     }
 
     companion object {
+        protected fun erBrukerBritiskBorgerUtenAnnetEøsStatsborgerskap(statsborgerskap: List<Statsborgerskap>, kontrollPeriodeForPersonhistorikk: Kontrollperiode): Boolean {
+            if (!statsborgerskap.erBritiskBorger(kontrollPeriodeForPersonhistorikk)) {
+                return false
+            }
+
+            return !statsborgerskap
```

Comment:
Hadde det vært mulig å skrive dette uten å bruke "!" ?

For min del så synes jeg det ble vanskelig å lese akkurat hva det er denne sjekker på

Reply:
Jeg kunne ha skrevet det som return statsborgerskap == false, men da foreslår IntelliJ å forenkle uttrykket til !statsborgerskap.